pubspec.yaml

```diff
@@ -1,15 +1,15 @@
 name: coverage
-version: 0.13.6
+version: 0.13.7
```

Comment:
[optional] Should we just bump this to 1.0.0?

Reply:
this isn't breaking - I would do it separately afterwords or wait for NNBD?

Reply:
Ah true. Let's just wait until NNBD.